src/datanode/src/datanode.rs

```diff
@@ -113,13 +113,16 @@ pub struct CompactionConfig {
     pub max_inflight_task: usize,
```

Comment:
I think these parameters should be in the plural. For example: `max_inflight_tasks`, `max_files_in_level0` and` max_purge_tasks`.